utilities/src/main/java/com/splicemachine/qpt/SQLSignature.java

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
Please add a description of the hash signature algorithm. Maybe some information on why it was chosen, if the signature is unique enough etc.

The same for Encoding.java. Is it possible maybe to chose another name for this class, that the name says which kind of encoding is used here?